starlette/requests.py

```diff
@@ -15,6 +15,10 @@
     parse_options_header = None
 
 
+if typing.TYPE_CHECKING:  # pragma: nocover
```

Comment:
IMO better to add directly in setup.cfg
```
[coverage:report]
exclude_lines =
    if typing.TYPE_CHECKING:
    @typing.overload
    ...
```
WDYT?

Reply:
Yeah makes sense. We'll probably need it in other places.